easybuild/easyconfigs/r/ReFrame-HPC/ReFrame-HPC-2.18.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PythonPackage'
+
+req_py_majver = 3
+req_py_minver = 5
+
+name = 'ReFrame-HPC'
```

Comment:
@vkarak Why not just `ReFrame`, which is the actual project name?

Reply:
The problem is that we were forced to use `reframe-hpc` as a name for the PyPI, cos there is another `reframe` project, completely unrelated... If there is a way to name to the easyconfig file and the resulting module `reframe`, it would be great for me, but I couldn't find a way to do this.

Reply:
Well, you can, but you just can't use the `PYPI_SOURCE` template.

Here's what I would do:

```python
name = 'ReFrame'
...
source_urls = ['https://pypi.python.org/packages/source/R/ReFrame-HPC']
```

Shouldn't need to change anything else.

(see also output of `eb -a --avail-easyconfig-templates | grep PYPI_SOURCE`)

Reply:
@boegel Coo, this is what I needed!